app/src/main/java/bisq/android/util/DateUtil.kt

```diff
@@ -36,14 +36,16 @@ class DateUtil : JsonDeserializer<Date> {
         private const val TAG = "DateDeserializer"
         private val LOCALE = Locale.US
         private const val PATTERN = "yyyy-MM-dd HH:mm:ss"
+        private val TIMEZONE = TimeZone.getDefault()
```

Comment:
@devinbileck These timezone tests proved to be trickier than I thought. For the record though I never got and I still don't get any failed tests.
But I'm not sure I understand your request.. If we remove that line then we cannot test `DateUtil.format()` anymore with timezone parameter which was the objective. Unless you mean to also replace the function's default argument line below, from
`timezone: TimeZone = TIMEZONE`
to
`timezone: TimeZone = TimeZone.getDefault()`
In which case the tests work and pass. But it's the same thing and no difference on my side, as the tests were already succeeding.
The `formatter.timeZone = timezone` needs to stay as it is though, otherwise the passed timezones from the tests will be ignored.
I don't fully understand what you mean, can you please explain?

Reply:
Apologies for not providing further details previously.

Yes, I was referring to replacing the argument's default value to:
`timezone: TimeZone = TimeZone.getDefault()`

When I run just the `DateUtilTest` class, all tests pass successfully. However, when I run the entire suite of tests I then encounter failures in `DateUtilTest`. I suspect it is a result of setting the `TIMEZONE` val within the companion object which obtains its value prior to DateUtilTest setting the default timezone. Which is why I mentioned setting the timezone argument default value.

When testing this, are you only running the `DateUtilTest` class?

Reply:
Okay I see what you mean now and you are absolutely right, I was only running the `DateUtilTest` class. Running the whole suite instead gives me some failed tests as well.
I suspected in the beginning that using `TimeZone.setDefault` would give weird issues like this which is why I tried to avoid it in my first commit regarding the tests, however now with your suggestion they all pass even when running the whole suite.